### ISSUE TITLE
ImageLayers cannot find tags for public image

### ISSUE THREAD

**Issue by pavel-paulau** [CenturyLinkLabs/imagelayers-graph#94]
Hi,

I added this public repository yesterday:

https://hub.docker.com/r/pavel/howdy/

However, ImageLayers cannot find image tags.

Steps:
1. Navigate to https://imagelayers.io/
2. Click on "Search for images on the Docker Hub".
3. Enter "pavel/howdy".

Outcome:
```
Request URL:https://imagelayers.io/registry/images/pavel/howdy/tags
Request Method:GET
Status Code:500 Internal Server Error
Remote Address:64.211.224.18:443

GET https://registry-1.docker.io/v1/repositories/pavel/howdy/tags returned 404
```

It seems like you are trying to use deprecated API.

**Comment by pavel-paulau** [comment]
```
$ curl -s https://registry.hub.docker.com/v2/repositories/pavel/howdy/tags/ | python -mjson.tool
{
    "count": 1,
    "next": null,
    "previous": null,
    "results": [
        {
            "creator": 3184,
            "full_size": 5271296,
            "id": 3332734,
            "image_id": null,
            "last_updated": "2016-06-10T01:17:04.137063Z",
            "last_updater": 3184,
            "name": "latest",
            "platforms": [
                5
            ],
            "repository": 731528,
            "v2": true
        }
    ]
}
```

**Comment by marco-roy** [comment]
I'm experiencing the same issue, but only for some images. It seems to only affect images that have been built more recently.

**Comment by streamnsight** [comment]
same here. Looks like the tags endpoint return a 404
is it the dockerHub that is slow to propagate changes? My images are 1-2 days old.

**Comment by streamnsight** [comment]
actually it looks like ImageLayer is still looking at a v1 registry, not v2

**Comment by geerlingguy** [comment]
Still an issue today :(

None of the images I've added in the last few months work on the service.

**Comment by wesm-outreach** [comment]
I'm also experiencing this issue
The following images could not be found: wesmoutreach/ssh:latest 
https://hub.docker.com/r/wesmoutreach/ssh/tags/

**Comment by gargamile** [comment]
Still an issue! I was excited to use Imagelayers, but it is still not able to communicate with Docker Hub v2

**Comment by cron410** [comment]
Still and issue for me too. packetworks/nxfilter-base and packetworks/nxfadmin

**Comment by yousefamar** [comment]
Has this really been broken for that long? I guess this repo is abandoned; that's a real shame.

**Comment by guikcd** [comment]
Same issue for a public image created today.

**Comment by gsauthof** [comment]
Yes, the imagelayers web service seems to be pretty much abandoned for some time.

Currently, MicroBadger is a viable alternative, for example:

- https://microbadger.com/images/packetworks/nxfilter-base
- https://microbadger.com/images/packetworks/nxfadmin
- https://microbadger.com/images/wesmoutreach/ssh:latest
- https://microbadger.com/images/pavel/howdy/

Also weird that docker.com isn't integrating such basic functionality into their web frontend.

**Comment by rupakg** [comment]
Hi Georg,
  Microscaling officially took over the ImageLayers project.

Thanks,
Rupak Ganguly

**Comment by wesm-outreach** [comment]
Thanks, I'm going to start using MicroBadger.

**Comment by guikcd** [comment]
Nice !
